services/121-service/src/payments/fsp-integration/intersolve-visa/dto/intersolve-block.dto.ts

```diff
@@ -12,7 +12,7 @@ export enum UnblockReasonEnum {
   UNBLOCK_GENERAL = 'UNBLOCK_GENERAL',
 }
 
-export class IntersolveBlockWalletResponseDto {
+export class IntersolveApiBlockWalletResponseDto {
   public data: IntersolveBlockWalletResponseBodyDto;
```

Comment:
This partial DTO is only used in this DTO. We agreed to refactor that to an inline part of the DTO and not in a separate class.